### PR TITLE
fix(ingest): bigquery - views in case more than 1 datasets with views

### PR DESCRIPTION
**Pull request by anshbansal**
Currently if we have BQ project with multiple datasets each having views only the first dataset's views were being extracted. This fixes that problem.

The method returned if there were any any views for the project. But it should only have returned if there were views for that particular dataset of the project.

## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by anshbansal** [comment]
Broken build for docs fix in https://github.com/datahub-project/datahub/pull/6997

File: metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py [hunk 1]
```diff
@@ -1231,15 +1231,13 @@ def get_views_for_dataset(
         project_id: str,
         dataset_name: str,
     ) -> List[BigqueryView]:
-        views = self.db_views.get(project_id)
+        views = self.db_views.get(project_id, {}).get(dataset_name, [])
 
         if not views:
```
[the remark this answers is not in the document]
Comment (anshbansal) [reply metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py:1236]:
```
Type "help", "copyright", "credits" or "license" for more information.
>>> a = []
>>> not a
True
>>> a = ['string1']
>>> not a
False
```

Empty array is considered `False` in Python. I would prefer to be more explicit. But did not want to make unrelated refactoring changes here other than the bug fix.

**Comment by gabe-lyons** [comment]
Are you planning on adding tests to prevent a regression from being introduced?

**Review by treff7es — APPROVED** [review]
lgtm

**Pull request merged**